components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -952,6 +1022,28 @@ def getAnnotation(self, ns=None):
             return AnnotationWrapper._wrap(self._conn, rv[0].child, link=rv[0])
         return None
 
+    def getAnnotationCount(self):
```

Comment:
Couldn't use `countAnnotations` because it's also defined in https://github.com/dominikl/openmicroscopy/blob/c99b0e5c0e7535326d947cc4f0e04666cf938234/components/tools/OmeroWeb/omeroweb/webclient/webclient_gateway.py#L2040 . I'll rename both methods to `getAnnotationCounts` to be consistent.